pkg/secret-store/service/k8s_secrets_service.go

```diff
@@ -131,20 +132,28 @@ func (secretsService *K8sSecretsService) DeleteSecret(name string) error {
 }
 
 func (secretsService *K8sSecretsService) UpdateSecret(secret models.Secret) (*models.Secret, error) {
-	var entities []*secretstore.SecretEntity
+	entity := secretstore.SecretEntity{}
+	name := *secret.Name
 
+<<<<<<< HEAD
```

Comment:
Fix the merge

Reply:
oh... It looks like my IDE does not automatically save after I tell it to take one change over another... I can fix this quickly.